src/main/java/seedu/address/logic/Messages.java

```diff
@@ -77,7 +77,7 @@ public static String format(Transaction transaction) {
                 .append("; Amount: ")
                 .append(transaction.getAmount())
                 .append("; Other party: ")
-                .append(transaction.getOtherParty())
+                .append(String.format(".2f", transaction.getOtherParty()))
```

Comment:
I think you changed otherParty instead of amount here

Reply:
Thanks for spotting the error.